### PR TITLE
Issue1634 conservation equation

### LINKED ISSUE #1634: Missing nominal value for mXi in ConservationEquation
This issue is to set the nominal attribute for `mXi` in `Buildings/Fluid/Interfaces/ConservationEquation.mo`, such as
```diff
-  Modelica.Units.SI.Mass[Medium.nXi] mXi(start=fluidVolume*rho_start*X_start[1:
-        Medium.nXi]) "Masses of independent components in the fluid";
+  Modelica.Units.SI.Mass[Medium.nXi] mXi(
+    start=fluidVolume*rho_start*X_start[1:Medium.nXi],
+    nominal=0.001*ones(Medium.nXi)) "Masses of independent components in the fluid";
```
The reason is that in Dymola 2023x beta2, the model `WetCoilDiscretizedMassFlow` fails to converge, and in earlier versions, the trajectory looks bad around t=2870:
image
The yellow line is from the simulation
```
simulateModel("Buildings.Fluid.HeatExchangers.Examples.WetCoilDiscretizedMassFlow", stopTime=3600, method="Cvode", tolerance=1e-06, resultFile="WetCoilDiscretizedMassFlow");
```
with a nominal value of 0.001.

### PR DESCRIPTION
**Pull request by mwetter**
This closes #1634

It changes the state values in the volume from `mXi` to `Medium.Xi` which allows setting the nominal attribute in Dymola without having to use a nonliteral value, in which case the assignment would be ignored.

### CODE REVIEW

**Comment by mwetter** [comment]
@Mathadon : Can you please review this change in state variable. This allows proper scaling of the state variable without using a non-literal equation.
This has already been merged to the master of the Buildings library.

File: IBPSA/Media/Air.mo [hunk 1]
```diff
@@ -1057,6 +1057,12 @@ if <i>T=0</i> &deg;C and no water vapor is present.
 </html>", revisions="<html>
 <ul>
 <li>
+September 9, 2022, by Michael Wetter:<br/>
+Set nominal attribute for <code>BaseProperties.Xi</code>.<br/>
+This is for
+<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1045\">#1634</a>.
```
Comment (Mathadon) [inline IBPSA/Media/Air.mo:1063]:
```suggestion
<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1634\">#1634</a>.
```

File: IBPSA/Media/Specialized/Air/PerfectGas.mo [hunk 2]
```diff
@@ -633,6 +635,12 @@ space dimension</i>. CRC Press. 1998.
 </html>", revisions="<html>
 <ul>
 <li>
+September 9, 2022, by Michael Wetter:<br/>
+Set nominal attribute for <code>BaseProperties.Xi</code>.<br/>
+This is for
+<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1045\">#1634</a>.
```
Comment (Mathadon) [inline IBPSA/Media/Specialized/Air/PerfectGas.mo:641]:
```suggestion
<a href=\"https://github.com/ibpsa/modelica-ibpsa/issues/1634\">#1634</a>.
```

**Review by Mathadon — APPROVED** [review]
@mwetter just 2 suggestions!

This is a quite limited set of changes for a relatively important change, nice!

**Pull request merged**